openssl/aes.go

```diff
@@ -473,3 +502,9 @@ func newCipherCtx(cipher C.GO_EVP_CIPHER_PTR, mode C.int, key, iv []byte) (C.GO_
 	}
 	return ctx, nil
 }
+
+func bigUint64(b []byte) uint64 {
+	_ = b[7]
+	return uint64(b[7]) | uint64(b[6])<<8 | uint64(b[5])<<16 | uint64(b[4])<<24 |
```

Comment:
Interesting... is the `_ = b[7]` here to eliminate bounds checks for the next line, for perf?

Reply:
Yep, see go.dev/issue/14808. I'll add a comment pointing to that issue.